### PR TITLE
[SYCLomatic]Enable the migration of NCCL_VERSION/NCCL_VERSION_CODE

### PR DESCRIPTION
**Pull request by wenhuiNi**
Signed-off-by: Ni, Wenhui <wenhui.ni@intel.com>

### CODE REVIEW

**Review by AndyCHHuang — APPROVED** [review]
LGTM

File: clang/test/dpct/nccl.cu [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 int check(ncclResult_t);
 
+// CHECK: #if DPCT_CCL_VERSION >= 0
+#if NCCL_VERSION_CODE >= NCCL_VERSION(2,13,0)
```
Comment (zhimingwang36) [inline clang/test/dpct/nccl.cu:10]: current NCCL_VERSION(X,Y,Z) it migrate to zero,  maybe not make sense in some case,  how about migrate to "DPCT_CCL_VERSION" an non zero value.

**Review by AndyCHHuang — APPROVED** [review]
LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**